File: Server/NoteServer/models/note.js
const mongoose = require('mongoose')

var Schema = mongoose.Schema;

const noteSchema = new Schema({
    id:{
        type: String
    },
    timestamp:{
        type: Date,
        default: Date.now
    },
    title: {
        type: String,
    },
    content: {
        type: String
    },
    fav: {
        type: Boolean,
        default: false
    }

});

module.exports = mongoose.model('Note', noteSchema);

File: Client/app/src/main/java/com/example/note/data/Note.kt
package com.example.note.data

import com.google.gson.annotations.SerializedName

data class Note(
    @SerializedName("_id") val id: String? = null,
    var timestamp: String? = null,
    var title: String? = null,
    var content: String? = null,
    var fav: Boolean = false
)
